pkg/services/sso/keycloak_service_proxy.go

```diff
@@ -10,13 +10,11 @@ import (
 	"github.com/openshift-online/ocm-sdk-go/authentication"
 )
 
-type tokenProvider interface {
-	GetToken() (string, error)
-}
+type tokenProvider func() (string, error)
```

Comment:
Why this change?

Reply:
Because the `tokenProvider` is just one method interface: I find creating an interface to pass an object with just one method to be much _javish_ and a bit overkill provided that in go we can pass functions.

Reply:
Thanks for the explanation.